ci/gpu/build.sh

```diff
@@ -43,10 +43,12 @@ nvidia-smi
 logger "Activate conda env..."
 source activate gdf
 conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/xgboost -c nvidia \
+      rapidsai/label/cuda${CUDA_REL}::cupy>=6.2 \
```

Comment:
You might want to quote the `>=6.2` for safety since I've seen strange things happen when the shell interprets the `>` char.